scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -14,7 +14,17 @@ import scala.meta.internal.semanticdb.Scala.{Descriptor => d}
 import scala.meta.internal.semanticdb.Scala.{Names => n}
 import scalafix.internal.v0._
 
-class SemanticdbPrinter(env: Env, index: DocumentIndex, config: RscCompatConfig) extends Printer {
+class SemanticdbPrinter(
+    env: Env,
+    addedImports: SyntheticScope,
+    index: DocumentIndex,
+    config: RscCompatConfig
+) extends Printer {
+
+  private val fullEnv = Env(addedImports +: env.scopes)
+
+  private def lookup(name: n.Name): String = fullEnv.lookup(name)
```

Comment:
I don't think that being able to abbreviate `fullEnv.lookup(name)` to `lookup(name)` is really worth adding an extra method, especially since the name is so generic.